src/PublicApiGeneratorTests/Namespaces_whitelisting.cs

```diff
@@ -32,12 +32,7 @@ public class Simple2
         [Fact]
         public void Should_filter_microsoft_namespace()
         {
-            AssertPublicApi(new[] { typeof(Simple1), typeof(Simple2) },
```

Comment:
I have to change 2 tests here to align with new behavior. I do not think that it should be treated as a breaking change. It's more like a bugfix since while/black-listing should work for entire classes as well.